pkg/libvmi/storage.go

```diff
@@ -40,6 +40,14 @@ func WithContainerDisk(diskName, imageName string) Option {
 	}
 }
 
+// WithContainerDiskAndPullPolicy specifies the disk name, the name of the container image and Pull Policy to be used.
+func WithContainerDiskAndPullPolicy(diskName, imageName string, imagePullPolicy k8sv1.PullPolicy) Option {
```

Comment:
You could make `WithContainerDisk` call `WithContainerDiskAndPullPolicy` with an empty pull policy, so code duplication is avoided.

Reply:
call ??
do you mean change the name  ??